pkg/rpc/loadbalancer/types.go

```diff
@@ -0,0 +1,95 @@
+package loadbalancer
+
+import (
+	"time"
+
+	"github.com/docker/infrakit/pkg/spi/loadbalancer"
+)
+
+// NameRequest is the rpc wrapper for the Name request
+type NameRequest struct {
```

Comment:
See my comment above. We can add packages when we need to. So what you have here is good!